### PR TITLE
Make CPU's float

### PR DESCRIPTION
**Pull request by PineappleIOnic**
This PR makes CPU's float so they work correctly with variable runtimes

### CODE REVIEW

**Review by christyjacob4 — APPROVED** [review]

File: app/http.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
 use Utopia\Http\Route;
 use Utopia\Http\Validator\Assoc;
 use Utopia\Http\Validator\Boolean;
+use Utopia\Http\Validator\FloatValidator;
```
Comment (Meldiron) [inline app/http.php:38]: Can we rename it to `Float`? It's already in namespace of `Validator`.

File: app/http.php [hunk 2]
```diff
@@ -711,15 +712,15 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
...
+    ->param('cpus', 1.0, new FloatValidator(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Container RAM memory.', true)
     ->param('version', 'v3', new WhiteList(['v2', 'v3']), 'Runtime Open Runtime version.', true)
     ->param('runtimeEntrypoint', '', new Text(1024, 0), 'Commands to run when creating a container. Maximum of 100 commands are allowed, each 1024 characters long.', true)
     ->inject('activeRuntimes')
     ->inject('response')
     ->inject('log')
     ->action(
-        function (string $runtimeId, ?string $payload, string $path, string $method, array $headers, int $timeout, string $image, string $source, string $entrypoint, array $variables, int $cpus, int $memory, string $version, string $runtimeEntrypoint, Table $activeRuntimes, Response $response, Log $log) {
+        function (string $runtimeId, ?string $payload, string $path, string $method, array $headers, int $timeout, string $image, string $source, string $entrypoint, array $variables, float $cpus, int $memory, string $version, string $runtimeEntrypoint, Table $activeRuntimes, Response $response, Log $log) {
```
Comment (Meldiron) [inline app/http.php:723]: We use $cpus in `setCpu()`. that is https://github.com/utopia-php/orchestration/blob/main/src/Orchestration/Orchestration.php#L217

Notice it accepts int. We need to change orchestration library to support float too

Comment (christyjacob4) [reply app/http.php:723]: Yeap we updated it 👍  Release 0.10.1 of it

File: app/http.php (hunk 1 above)
Comment (christyjacob4) [reply app/http.php:38]: its actually called FloatValidator in the http library 
https://github.com/utopia-php/http/blob/master/src/Http/Validator/FloatValidator.php

We need to fix it there. 

For now we can do `use Utopia\Http\Validator\FloatValidator as Float;`

Comment (abnegate) [reply app/http.php:38]: We can't do either (I wanted to do this previously 😅) Class names are case-insensitive in PHP, and classes named 'int', 'string', 'float', 'bool', 'true', 'false', or 'null' are forbidden

**Pull request merged**